content/en/docs/setup/getting-started/index.md

```diff
@@ -9,46 +9,30 @@ aliases:
 keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
-This guide is intended for users who are new to Istio and lets you quickly evaluate Istio by installing the
-`demo` [configuration profile](/docs/setup/additional-setup/config-profiles/).
+This guide lets you quickly evaluate Istio.
 
-If you are already familiar with Istio or interested in installing other configuration profiles
-or a more advanced [deployment model](/docs/ops/deployment/deployment-models/),
-follow the [installing with {{< istioctl >}} instructions](/docs/setup/install/istioctl) instead.
+These steps assume you are using
+[Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) and have a
```

Comment:
Assuming minikube doesn't sound like a simplification. Minikube is not the quickest way to get started unless minikube is already installed. Creating a cluster using Docker Desktop or IBM Cloud (and I assume other clouds like GCE) is an easier way to start from scratch than installing and setting up minikube and minikube tunnel.

These instructions should just say you need to start with a Kubernetes cluster (it can mention minikube as one way to do it), but assuming minikube throughout the instructions is not a good idea IMO.

Reply:
I don't know that IBM Cloud is unless they also have an account and the CLI installed. Similar for GKE and others. I'm not sure if they would already have Docker installed (we ask our developers to use it, but potential uses kicking the tires?).

Reply:
@ericvn Yes, that sounds good. In fact, that's similar to what linkerd does in their quick start. They just say that you need a kubernetes cluster and they point to minikube and docker desktop as a couple of easy choices if you're also new to Kubernetes.

Regarding IBM Cloud or Docker Desktop being easier than minikube, I think that a good percentage of the people that are looking at Istio for the first time are not also new to Kubernetes and Docker. I'm assuming that many of the people trying to get started with Istio already use Kubernetes, and/or Docker, so they probably already have something like IBM Cloud or Docker Desktop or Minikube configured. So we can't say that regardless of what may be the easiest way to set up a cluster in you case, these instructions are specifically for minikube, so if you want to get started quickly with Istio you need to install minikube first.

Reply:
Users were failing to successfully complete the old Getting Started and the feedback was asking us to ensure we had the exact successful steps documented for a particular install path, minikube being the one that's free and accessible to everyone as our common starting point.

But we add note at the beginning the platform setup link and provide the "Other" tab at the mid-point so someone can still be successful using IBM or GKE if they prefer. Maybe we should just rename the Other tab to be more specific about IBM cloud, GKE, etc?

Reply:
I see, so the instructions don't really "assume you are using minikube". The instructions work for any platform, it's just that minikube is the fast path. I guess that's OK, but you should reword the above sentence to just say that the user needs a Kubernetes cluster to get started, and mention minikube as a good choice for somebody also new to k8s. Then you can make minikube the simplest option in the following instructions, just so long as the instructions also work well on the other platforms too

Reply:
fixed